presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -128,4 +132,85 @@ public String toString()
         limit.ifPresent(value -> builder.append(" limit=").append(value));
         return builder.toString();
     }
+
+    public static Builder builder()
+    {
+        return new Builder();
+    }
+
+    public static Builder builderFrom(JdbcTableHandle handle)
+    {
+        return new Builder(handle);
+    }
+
+    public static final class Builder
+    {
+        private SchemaTableName schemaTableName;
+        @Nullable
```

Comment:
Can you please use `Optional`? Notice that after introducing builder it is very easy to modify the constructor of `JdbcTableHandle` to accept `Optional`s also.

Reply:
> it is very easy to modify the constructor of JdbcTableHandle to accept Optionals also.

I'm sorry, but I couldn't understand it. Do you mean?
```java
    public JdbcTableHandle(SchemaTableName schemaTableName, Optional<String> catalogName, ...)
    {
        this(schemaTableName, catalogName.orElse(null), ...);
    }
```